aggregator/options.go

```diff
@@ -869,8 +874,20 @@ func (o *options) computeFullGaugePrefix() {
 }
 
 // By default we use e.g. ".p50", ".p95", ".p99" for the 50th/95th/99th percentile.
-func defaultTimerQuantileSuffixFn(quantile float64) []byte {
-	return []byte(".p" + strconv.FormatFloat(quantile*100, 'f', -1, 64))
+func defaultTimerQuantileSuffixFn(quantile float64) ([]byte, error) {
+	if quantile < 0 || quantile > 1 {
+		return nil, fmt.Errorf("invalid quantile %f", quantile)
+	}
+	str := strconv.FormatFloat(quantile*100, 'f', -1, 64)
+
+	if strings.Contains(str, ".") {
```

Comment:
nit: Can directly call `strings.Index(str, ".")` here and check it against -1.

Reply:
sg